metadata.yaml

```diff
@@ -8,4 +8,10 @@ tags:
 requires:
   reverseproxy:
     interface: http
-series: ['xenial']
\ No newline at end of file
+resources:
+    dhparam:
+        type: file
+        filename: dhparam.pem
```

Comment:
Is there a reason why the charm can't generate the dhparam file itself? Seems like way to low-level to expose to the operator.